source/js/main.js

```diff
@@ -55,8 +55,12 @@ let main = {
     Array.from( document.querySelectorAll(`a`) ).forEach((link) => {
       let href = link.getAttribute(`href`);
 
+      function checkDomain(domain) {
+        return !href.match(`//${domain}`);
+      }
+
       // Define an external link as any URL with `//` in it
-      if (href && href.match(/\/\//) && !href.match(`//${env.TARGET_DOMAIN}`)) {
+      if (href && href.match(/\/\//) && env.TARGET_DOMAINS.some(checkDomain)) {
```

Comment:
if `checkDomain` is just a one-liner, we might as well use this:
```
if (href && href.match(`//`) && env.TARGET_DOMAINS.some(href => !href.match(`//${domain}`)) {
  ...
```
and dispense with the `checkDomain` function on L58.